salt/modules/useradd.py

```diff
@@ -927,3 +927,111 @@ def _getpwall(root=None):
             # Generate a getpwall compatible output
             comps[2], comps[3] = int(comps[2]), int(comps[3])
             yield pwd.struct_passwd(comps)
+
+
+def add_subuids(name, first=100000, last=110000):
+    '''
+    Add a range of subordinate uids to the user
+
+    name
+        User to modify
+
+    first
+        Begin of the range
+
+    last
+        End of the range
+
+    CLI Examples:
+
+    .. code-block:: bash
+
+        salt '*' user.add_subuids foo
+        salt '*' user.add_subuids foo first=105000
+        salt '*' user.add_subuids foo first=600000000 last=600100000
+    '''
+    if __grains__['kernel'] != 'Linux':
+        log.warning("'subuids' are only supported on GNU/Linux hosts.")
```

Comment:
The warnings you've added in each of these functions are unnecessary, as this module will not be loaded on non-Linux minions (see [here](https://github.com/saltstack/salt/blob/501079fffc95f50f338d50eaa9dfd1078c8bc0c5/salt/modules/useradd.py#L40-L47)).

The warnings in each of the new functions you've added should be removed.

Reply:
Oh, disregard... I was under the impression that pw was used on the other BSD variants, and AIX was also added to the list of OSes which uses this module.

My bad!